common-lib/src/main/scala/com/gu/mediaservice/model/Collection.scala

```diff
@@ -0,0 +1,18 @@
+package com.gu.mediaservice.model
+
+import org.joda.time.DateTime
+import play.api.libs.json._
+
+case class Collection(path: List[String], actionData: actionData)
+object Collection {
+  implicit def formats: Format[Collection] = Json.format[Collection]
+}
+
+// Following the crop structure
+// TODO: Use this in crop too
+case class actionData(author: String, date: DateTime)
+object actionData {
+  implicit def formats: Format[actionData] = Json.format[actionData]
+  implicit val dateWrites = Writes.jodaDateWrites("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
+  implicit val dateReads = Reads.jodaDateReads("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
```

Comment:
I've updated this so that we always [write with millis](https://github.com/guardian/grid/commit/d230bb8e8d0a4be8a98ec1090bfa69d7f4c3b56a#diff-7e88df982621cdb0d9fcefd26e742452R10) even if we didn't have them.

When we read - we use [two parsers](https://github.com/guardian/grid/blob/d230bb8e8d0a4be8a98ec1090bfa69d7f4c3b56a/common-lib/src/main/scala/com/gu/mediaservice/lib/formatting/package.scala#L14-L15) so we should be good on both fronts.

I'd like to do a little more testing though to see if this screws up anything on the client (I can't image it would).